spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/TaskControllerTests.java

```diff
@@ -228,10 +234,13 @@ public void testLaunch() throws Exception {
 
 	@Test
 	public void testLaunchWithAppProperties() throws Exception {
+
 		repository.save(new TaskDefinition("myTask2", "foo2 --common.prop2=wizz"));
 		this.registry.register("task.foo2", new URI("file:src/test/resources/apps/foo-task"));
 
-		mockMvc.perform(post("/tasks/executions").param("name", "myTask2").accept(MediaType.APPLICATION_JSON))
+		mockMvc.perform(post("/tasks/executions").param("name", "myTask2")
+				.contentType(MediaType.APPLICATION_FORM_URLENCODED)
```

Comment:
If you leave out the content type, does it still work?  I ask because we said we didn't want to break the API, yet I don't see the tests for the old way still here.

Reply:
It should work. I will remove the contentType to default to the previous test